pywbem/_recorder.py

```diff
@@ -279,15 +285,18 @@ def open_file(filename, file_mode='w'):
         done differently in pythong 2 and python 3
 
         Parameters:
-          filename: (:term: `string`):
+
+          filename(:term:`string`):
             Name of the file where the recorder output will be written
-          file_mode: (:term: `string`):
+
+          file_mode(:term:`string`):
             Optional file mode.  The default is 'w' which overwrites any
             existing file.  if 'a' is used, the data is appended to any
             existing file.
 
-          Example:
-            recorder = TestClientRecorder(open_file('recorder.log')
+        Example::
+
+            recorder = TestClientRecorder(open_file('recorder.log'))
```

Comment:
The example does not work, because `open_file()` as a static method still needs either an object or the class to be invoked on. Change `open_file(...)` to `TestClientRecorder.open_file(...)`.